docs/manual/guide/advanced/IntegratingNonLagom.md

```diff
@@ -0,0 +1,57 @@
+# Integrating with non Lagom services
+
+## Invoking Lagom services
+
+Lagom service calls are implemented using idiomatic REST.  The simplest way to invoke a Lagom service from another framework is to use that frameworks REST client to invoke the Lagom service.
+
+Another way to implement Lagom services, if the client is running in a JVM, is to use the Lagom service interface directly.
+
+### Using the Lagom service client
+
+#### Configuring dependencies
+
+To use the Lagom service interface, you will need to add a dependency on the Lagom integration client to your build.  If using maven, this can be done by adding the following dependency:
+
+```xml
+    <dependency>
+        <groupId>com.lightbend.lagom</groupId>
+        <artifactId>lagom-javadsl-integration-client_2.11</artifactId>
+        <version>${lagom.version}</version>
+    </dependency>
+```
+
+In addition you will also need to add a dependency to the API artifact for your Lagom service to your project.
```

Comment:
I find the text here not straightforward and feel that people may not immediately see what is the API artifact (they may wonder if that is Lagom javadsl-api artifact).